### ISSUE TITLE
Logits as argument to reduce_mean functions 

### ISSUE THREAD

**Issue by danyz91** [ponderousmad/ud730#2]
Hi,
I think i found an issue in the code, in the "Classifier" code section under Problem 2 Markdown. Here it is:
`# Classifier.`
 `           predicted_embeddings = tf.nn.xw_plus_b(tf.concat(0, outputs), w, b)
            loss = tf.reduce_mean(tf.nn.softmax_cross_entropy_with_logits(logits, tf.concat(0, train_labels)))`

I'm pretty sure that predicted_embeddings must be a parameter for reduce_mean function instead of 'logits'
The evidences for my issue are:
- predicted_embeddings never used
- logits not initialized

Tell me if it is correct of if i make a mistake on my own
Bye
